.github/ISSUE_TEMPLATE/bug.yml

```
@@ -5,6 +5,26 @@ name: Bug
 title: "<SCOPE>: <SUMMARY>"
 
 body:
+  - type: "checkboxes"
+
+    attributes:
+      label: "I assert that this issue is relevant for Stylix"
+      description: |
+        This bug tracker is for actionable issues that are not the result of
+        user error. If you need help using your system and are unsure if this
+        is a bug with Stylix, please consider reading the
+        [Stylix Book](https://stylix.danth.me/) or asking for help in a
+        [Discussion](https://github.com/danth/stylix/discussions) or the
+        [Stylix Matrix Space](https://matrix.to/#/#stylix:danth.me) before
+        opening an issue.
```

What about converting the "user error" part into a checkbox and using the following adapted phrasing:

```suggestion
        If you need help or are unsure whether this is a Stylix bug, please
        consider reading the [Documentation](https://stylix.danth.me) or asking
        for help in a [GitHub Discussion](
        https://github.com/danth/stylix/discussions) or the [Stylix Matrix
        Space](https://matrix.to/#/#stylix:danth.me) before opening an issue.
```

---

i like your altered phrasing, though i'm not sure that we need another checkbox as there is already "I assert that this is a bug and not a support request."

---

Good point. Maybe use:

> - [ ] I assert that this is a bug and not a user error or support request.